spec/models/ext_management_system_spec.rb

```diff
@@ -1,4 +1,24 @@
 RSpec.describe ExtManagementSystem do
+  SUPPORTED_MAPPING_ENTITIES = {
+    "ContainerGroup"      => "ContainerGroup",
+    "ContainerProject"    => "ContainerProject",
+    "ContainerNode"       => "ContainerNode",
+    "ContainerReplicator" => "ContainerReplicator",
+    "ContainerRoute"      => "ContainerRoute",
+    "ContainerService"    => "ContainerService",
+    "ContainerBuild"      => "ContainerBuild",
+    "VmVmware"            => "ManageIQ::Providers::Vmware::InfraManager::Vm",
+    "Vm"                  => "ManageIQ::Providers::Amazon::CloudManager::Vm",
+    "Image"               => "ManageIQ::Providers::Amazon::CloudManager::Template",
+    "VmAzure"             => "ManageIQ::Providers::Azure::CloudManager::Vm",
+    "VmIBM"               => "ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm",
+    "VmOpenstack"         => "ManageIQ::Providers::Openstack::CloudManager::Vm"
+  }.freeze
+
+  it "supports label mapping for provider subclasses" do
+    expect(SUPPORTED_MAPPING_ENTITIES).to eq(ExtManagementSystem.entities_for_label_mapping)
```

Comment:
Sorry @lpichler (just noticed you added this) but can you update this to check `.includes` on one or two values instead of `.eq` on the whole list?  I don't want to get back into having the same problems as we fixed in https://github.com/ManageIQ/manageiq/pull/20595 aka everytime we add another one core specs start failing.

You don't need to rerun the whole cross repo tests again though :)

Reply:
> Sorry @lpichler (just noticed you added this) but can you update this to check `.includes` on one or two values instead of `.eq` on the whole list? I don't want to get back into having the same problems as we fixed in #20595 aka everytime we add another one core specs start failing.

Done 👍 

> 
> You don't need to rerun the whole cross repo tests again though :)

👍  :)

thanks @agrare